release-notes/SupportedFeatures-v1.0.0-rc1.md

```diff
@@ -28,9 +28,9 @@ Supported features
 ||NetHttpBinding|:white_check_mark: | :white_check_mark: | :white_check_mark: | :warning: [more...](https://github.com/dotnet/wcf/issues/534) |
 ||NetTcpBinding|:white_check_mark: | :white_check_mark: | :white_check_mark: | :warning: [more...](https://github.com/dotnet/wcf/issues/534) |
```

Comment:
We will need a query for RC1 only bugs. The issues status reflects the latest status,